### PR TITLE
Set explicit version for the typing packages

### PR DESCRIPTION
**Pull request by bensze01**
## Description

Hotfix for the issues encountered in the internal and Open CIs. Should help prevent new issues introduced by updating these packages. Fixes https://github.com/Mbed-TLS/mbedtls/issues/8250

## PR checklist

- [ ] **changelog** not required
- [ ] **backport** not required
- [ ] **tests** not required

### CODE REVIEW

**Comment by gilles-peskine-arm** [comment]
That's probably a better fix than https://github.com/Mbed-TLS/mbedtls/pull/8248. Are types-xxx package versions aligned with xxx versions?

**Comment by bensze01** [comment]
They should be for jsonschema. For jinja2' the alignment of version numbers happened on a version that's more recent than what we require -so I just used the newest version our CI was already grabbing.

File: scripts/driver.requirements.txt [hunk 1]
```diff
@@ -14,6 +14,6 @@ markupsafe < 2.1
 Jinja2 >= 2.10.1; python_version <  '3.10'
 Jinja2 >= 2.10.3; python_version >= '3.10'
 # Jinja2 >=2.10, <3.0 needs a separate package for type annotations
-types-Jinja2
+types-Jinja2 >= 2.11.9
 jsonschema >= 3.2.0
-types-jsonschema
+types-jsonschema >= 3.2.0
```
Comment (mpg) [inline scripts/driver.requirements.txt:19]: I'm sorry, I don't get it. I thought the breakage was from pip picking up a version of types-jsonschema that's too recent for our version of Python. How does a `>=` dependency fix that?

To be clear: I'm not saying this is wrong, just really that I'm not getting it without some additional explanation.

Comment (bensze01) [reply scripts/driver.requirements.txt:19]: `min_requirements.py` transforms all `>=` requirements into `==` requirements' to make sure we don't depend on features introduced in later versions.

Comment (mpg) [reply scripts/driver.requirements.txt:19]: Ah, thanks for the reminder! Indeed this makes sense now :)

**Review by mpg — APPROVED** [review]
LGTM if the CI agrees.

**Review by gilles-peskine-arm — APPROVED** [review]
LGTM

**Comment by gilles-peskine-arm** [comment]
This fully fixes https://github.com/Mbed-TLS/mbedtls/issues/8250. The result analysis step is failing, but that's an independent problem, fixed by #8247.

The OpenCI run has finished. I'm merging without waiting for internal CI.

**Pull request merged**